### DISCUSSION TITLE
Strange effect on images

### DISCUSSION THREAD

**Discussion by popescualinflorin** [cotes2020/jekyll-theme-chirpy#1714] [Q&A] [answered]
### Checklist

- [x] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

hello

i used the code from "Text and Typography" to add images like "![Desktop View](/23052023/step1.png){: width="972" height="589" .w-75 .normal}"
but every image apper with somthing like this:
https://github.com/cotes2020/jekyll-theme-chirpy/assets/46263308/56324716-a335-493d-b8b9-bdeef0a5178f

there is a strange line that pass over the image

### Operations you have already tried

I tested it on different post, re-downloaded the template, tested on different browser
On the HTML site i notice that  class=" img-link w-50 right shimmer" removing shimmer was solving the problem, but I don't know how to solve it on all the posts

**Comment by huanyushi** [comment, marked as answer]
The OP didn't offer any source code. The only thing I know is that when the link address of your image is wrong, it will always have this loading animation. So the OP should check whether the image address is wrong.
<img src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/100574401/c7e7e800-5135-4417-bc69-873e3ec22aed" width=30%>

This animation effect is generally used to remind users that the page is loading images. The relevent settings can be seen here: [shimmer](https://github.com/search?q=repo%3Acotes2020%2Fjekyll-theme-chirpy%20shimmer&type=code). If you want to cancel this animation, you can add the following code to `assets/css/jekyll-theme-chirpy.scss`:

```scss
.shimmer::before{
  background:none;
}
```

**Reply by popescualinflorin** [reply]
thanks a lot i fount the shimmer under _sass/addons/commons.scss and removed the part that you indicated and it solved the problem.
Thanks again

**Reply by huanyushi** [reply]
@popescualinflorin  I think the best way is to overwrite the code of the original theme instead of deleting it. This will also make it easier to follow theme updates. As I said above, add code in `assets/css/jekyll-theme-chirpy.scss` may be a better choice:

```scss
.shimmer::before{
    content:""
}
```

(Just a suggestion...)

**Reply by elias-helou** [reply]
Creating the file `assets/css/jekyll-theme-chirpy.scss` does not seem to work for me. I am placing some content in my home page adding `{{ content }}` to the home template in my `home.html` layout (in the `_layouts` folder) and Jekyll keeps putting the image inside a `div` with the `shimmer` class. I'd rather change this behaviour than to change the shimmer behavior, but I can't figure out how to do this.

**Reply by seanthegeek** [reply]
The shimmer is showing up on all images even after they are fully loaded.

**Reply by klaas-sysop** [reply]
tried this but didnot work

assets/css/jekyll-theme-chirpy.scss 

.shimmer::before{
    content:""
}

**Reply by huanyushi** [reply]
> The shimmer is showing up on all images even after they are fully loaded.

Hi @seanthegeek, I have visited your website, but I didn't see the shimmer effect after loading images. Could you give me more details of your problem?

**Reply by huanyushi** [reply]
@klaas-sysop What about this? It can work for me.

```scss
.shimmer::before{
  background:none
}
```

**Reply by klaas-sysop** [reply]
@huanyushi thnx it worked

**Reply by huanyushi** [reply]
@klaas-sysop I have edited the original answer. And I have seen related issues, maybe that problem was caused by the recently version update. It may need to be fixed by the owner.

**Reply by Amrlxyz** [reply]
@huanyushi Do i just create a new file with the 3 lines of code? Doesn't seem to be working for me. Do i need to change any other config because the file `assets/css/jekyll-theme-chirpy.scss` isn't there previously.

**Reply by seanthegeek** [reply]
Actually, do not use that workaround. There are other bugs in 7.2.0 that workaround won't fix. Instead, downgrade to 7.1.1 until a fixed version is available by following the instructions [here](https://github.com/cotes2020/jekyll-theme-chirpy/issues/2068#issuecomment-2508177881).

**Reply by felmoltor** [reply]
Finding the same behaviour in 7.3.
For the workaround modifying `assets/css/jekyll-theme-chirpy.scss`, do I need to set the assets to self_host in _config.yml?
It's not working for me for now. And the effect is distracting.
